app/decorators/metric_decorator.rb

```diff
@@ -12,7 +12,7 @@ def info
 
   def info_content(status)
     h.concat h.content_tag(:i, nil, class: status_fa_icon(status))
-    h.concat h.content_tag(:span, "#{object.name} - #{object.value}")
+    h.concat h.content_tag(:span, "#{object.name} - #{object.value.to_f.round(2)}")
```

Comment:
This will fail when we start using metrics that are not numbers. But we can change it later and create an issue for this